src/stcal/jump/twopoint_difference.py

```diff
@@ -255,6 +281,52 @@ def find_crs(dataa, group_dq, read_noise, rejection_thresh,
                             if (gdq[integ, group, row, col + 1] & dnu_flag) == 0:
                                 gdq[integ, group, row, col + 1] =\
                                     np.bitwise_or(gdq[integ, group, row, col + 1], jump_flag)
+
+        # flag n groups after all jumps to account for the transient seen
+        flag_dn_threshold = [after_jump_flag_e1, after_jump_flag_e2]
+        flag_groups = [after_jump_flag_n1, after_jump_flag_n2]
+        # ensure the smallest threshold is 1st
+        # if flag_dn_threshold[0] > flag_dn_threshold[1]:
+        #     flag_dn_threshold = np.flip(flag_dn_threshold)
+        #     flag_groups = np.flip(flag_groups)
+
+        cr_group, cr_row, cr_col = np.where(np.bitwise_and(gdq[integ], jump_flag))
+
+        # **should** work and be faster.  But does not.  Sigh.
+        # for jj in range(2):
+        #     if flag_groups[jj] > 0:
+        #         uplimit = 1e6
+        #         if jj == 0:
+        #             if flag_groups[1] > 0:
+        #                 uplimit = flag_dn_threshold[1]
+        #         cdn_jumps = dn_jump[cr_group, cr_row, cr_col]
+        #         flagcrs = (cdn_jumps >= flag_dn_threshold[jj]) & (cdn_jumps < uplimit)
+        #         log.info(f"Flagging {flag_groups[jj]} groups after {flag_dn_threshold[jj]}
+        #                   DN jumps for {np.sum(flagcrs)} pixels.")
+        #         for group, row, col in zip(cr_group[flagcrs], cr_row[flagcrs], cr_col[flagcrs]):
+        #             for kk in range(group, min(group + flag_groups[jj], ngroups)):
+        #                 if (gdq[integ, kk, row, col] & sat_flag) == 0:
+        #                     if (gdq[integ, kk, row, col] & dnu_flag) == 0:
+        #                         gdq[integ, kk, row, col] =\
+        #                             np.bitwise_or(gdq[integ, kk, row, col], jump_flag)
+
+        for cthres, cgroup in zip(flag_dn_threshold, flag_groups):
+            if cgroup > 0:
+                log.info(f"Flagging {cgroup} groups after detected jumps with e >= {np.mean(cthres)}.")
+
+                for j in range(len(cr_group)):
+                    group = cr_group[j]
+                    row = cr_row[j]
+                    col = cr_col[j]
+                    if dn_jump[group - 1, row, col] >= cthres[row, col]:
+                        # print("stcal ", group, group + cgroup + 1,
+                        #       dn_jump[group - 1, row, col], cthres[row, col])
+                        for kk in range(group, min(group + cgroup + 1, ngroups)):
+                            if (gdq[integ, kk, row, col] & sat_flag) == 0:
+                                if (gdq[integ, kk, row, col] & dnu_flag) == 0:
+                                    gdq[integ, kk, row, col] =\
+                                        np.bitwise_or(gdq[integ, kk, row, col], jump_flag)
```

Comment:
the pr description says that pixels after a jump are flagged with 'do not use', but here they are being flagged with jump. 

one thought i have about this is that even though subsequent groups arent really 'jumps', neither are adjacent pixels but those obtain a jump flag too, so maybe it does make sense to use the jump flag here?

Reply:
Fixed the PR description.  Was wrong.

Yes, following what has been done previously, hence flagging as jumps.